### ISSUE TITLE
Minified exceptions are not usable

### ISSUE THREAD

**Issue by slokhorst** [zxing-js/library#342]
When trying to decode an image using the minified library at `https://unpkg.com/@zxing/library@latest`, I get the following exception:

```
s
    at g.f.decode (https://unpkg.com/@zxing/library@latest:1:15248)
    at f._.decodeBitmap (https://unpkg.com/@zxing/library@latest:1:30173)
    at f._.decode (https://unpkg.com/@zxing/library@latest:1:29472)
    at o (https://unpkg.com/@zxing/library@latest:1:28777)
    at https://unpkg.com/@zxing/library@latest:1:28992
    at new Promise (<anonymous>)
    at f._.decodeOnce (https://unpkg.com/@zxing/library@latest:1:28959)
    at f._.decodeFromImageElement (https://unpkg.com/@zxing/library@latest:1:25995)
    at f._.decodeFromImage (https://unpkg.com/@zxing/library@latest:1:25284)
```

I have no idea what is happening or how to catch or debug this.

Now when using the full library with the same image, I get a useful exception:
```
NotFoundException {message: undefined}
```

How am I supposed to use the exception thrown by the minified library?

**Comment by zacharytyhacz** [comment]
Hello @slokhorst , from what I can tell the `NotFoundException` means that no code was found in what you were decoding. 

The error being thrown from the minified unpkg build _seems_ to be due to not having the source maps and showing a minified and renamed `NotFoundException` error.

**Comment by slokhorst** [comment]
Hey @zacharytyhacz I think you're right.

I would imagine that the `NotFoundException` isn't supposed to be renamed. How am I supposed to catch or handle it when I don't know what it's called?

**Comment by zacharytyhacz** [comment]
@slokhorst 
> I would imagine that the `NotFoundException` isn't supposed to be renamed. How am I supposed to catch or handle it when I don't know what it's called?

Yes, it should not be renamed after being minified lol. this may be solved in this packages source with a minifying option to include source maps somewhere.

I can understand certain cases when you want to check the specific name of the error, even just in the console seeing what errors you're getting.

However, if you isolate where you call the decoding function, any error it throws means that no result was found. If no error is thrown, something was decoded and you have a true result.

If you have other functionality that could throw errors in your `try-catch` block, I can understand why you should specifically look for a `NotFoundException` error name

```javascript
try {
   const result = await codeReader.decodeFromImage(image);
   console.log("if it get's here, there's a true result:", result);
}catch(err){
   // no result found in the image
}
```

**Comment by odahcam** [comment]
Hello guys, @zacharytyhacz is actually pretty accurate! The [latest version](https://unpkg.com/browse/@zxing/library@0.18.2/umd/) now includes source maps for the minified bundle, which should help with this. In the future this method will stop throwing unecessary exceptions.
